src/ruby/puppetserver-lib/puppet/server/puppet_config.rb

```diff
@@ -86,7 +86,7 @@ def self.initialize_puppet(puppet_config:)
       Puppet.push_context(dummy_ssl_context)
     end
 
-    Puppet.settings.use :main, :master, :ssl, :metrics
+    Puppet.settings.use :main, :server, :ssl, :metrics
```

Comment:
The warning was specifically coming from here, but I updated the two fixture files as well for good measure.